documentation/api/query/v4/facts.markdown

```diff
@@ -80,7 +80,7 @@ Subquery against `/fact-contents` to get all remotely-authenticated trusted fact
 
     curl -X GET http://localhost:8080/v4/facts --data-urlencode 'query=["in", ["name","certname"],
       ["extract",["name","certname"],
-        ["select-fact-contents", ["~>", "path", [".*", "authenticated"]]]]]'
+        ["select_fact_contents", ["~>", "path", [".*", "authenticated"]]]]]'
```

Comment:
Its an interesting problem, since we've really focused on what the API returns more than what is passed in in regards to underscore/hyphens. I mean, if we focus on the user problems, its related to JS usage of returned data. In the future we may see more usage of passing in the endpoint names in a query however, not just in the endpoint that is ... if we wanted to start doing single endpoint style queries. This even affects PQL to some extent. I dunno, the OCD person in me says underscores in endpoints, but I didn't expect we'd want to do this just yet.

In a way I'd be fine sticking with select-foo-bar ... for the sake of getting this patch in, and coming back to this discussion.